test/parallel_api/numeric/numeric.ops/in_place_scan.pass.cpp

```diff
@@ -0,0 +1,98 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#define _GLIBCXX_USE_TBB_PAR_BACKEND 0 // libstdc++10
+
+#include "support/test_config.h"
+
+#include <cassert>
+
+#include _PSTL_TEST_HEADER(execution)
+#include _PSTL_TEST_HEADER(algorithm)
+
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+#    include "support/utils_sycl.h"
+#    include "support/sycl_alloc_utils.h"
+#endif
+
+
+#include <algorithm>
+#include <vector>
+#include <iostream>
+
+#include <numeric> // std::inclusive_scan, exclusive_scan
+#include <functional>
+
+int main()
+{
+
+#if TEST_DPCPP_BACKEND_PRESENT
+  std::vector<int> v{3, 1, 4, 1, 5, 9, 2, 6};
```

Comment:
How about using the existing scan test? Copy input to output first, and then run in-place scans on both sequences.

Reply:
Yes, it makes sense. Additionally, I would keep `in_place_scan.pass.cpp` as well because it is important to cover that case which came from a customer to be sure that it not be broken in the future.